boltdesignsystem/bolt
# DS-273 Remove 100% height from icon component

## Jira

https://pegadigitalit.atlassian.net/browse/DS-273

## Summary

Fixes a bug where the icon height is as tall as page height.

## Details

Removed a line in css that's making the icon container take up 100% height.

## How to test

Run the branch locally and check the follow pages where icons are used most:

1. Icon docs
2. Product pages
3. Academy completed training page

Then create some dummy code to check if it breaks flex display:

```
.container {
    display: flex;
    flex: 0 1 100%;
    flex-direction: column;
}

<div class="container">
  <bolt-icon name="chevron-up" size="large" background="circle" color="yellow"></bolt-icon>
  <bolt-icon name="chevron-up" size="large" background="circle" color="yellow"></bolt-icon>
</div>
```
